EmpleoDotNet/Views/Shared/_JobListPartial.cshtml

```diff
@@ -9,7 +9,7 @@
             <a href="@Url.Action("Detail", "JobOpportunity" , new { id=UrlHelperExtensions.SeoUrl(jobOpportunity.Id, jobOpportunity.Title)})" class="featured">
                 <div class="row">
                     <div class="col-md-1 hidden-sm hidden-xs text-center">
-                        @if (!string.IsNullOrEmpty(jobOpportunity.CompanyLogoUrl))
+                        @if (UrlHelperExtensions.IsImageAvailable(jobOpportunity.CompanyLogoUrl))
```

Comment:
@Willjobs94 we can make a migration and do it, but I think we need more feedback before.